explore/src/main/scala/explore/targets/TargetSearchResult.scala

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.targets
+
+import cats.Eq
+import cats.syntax.all._
+import explore.model.TargetWithOptId
+import japgolly.scalajs.react.Reusability
+import lucuma.catalog.AngularSize
+import lucuma.catalog.CatalogTargetResult
+import lucuma.core.model.Target
+
+final case class TargetSearchResult(
+  targetWithOptId: TargetWithOptId,
+  angularSize:     Option[AngularSize]
+) {
+  lazy val optId: Option[Target.Id] = targetWithOptId.optId
```

Comment:
Why lazy?

Reply:
So as not to allocate it unless it's needed.